src/rest_framework_api_key/permissions.py

```diff
@@ -19,17 +19,16 @@ def get(self, request: HttpRequest) -> typing.Optional[str]:
         return self.get_from_authorization(request)
 
     def get_from_authorization(self, request: HttpRequest) -> typing.Optional[str]:
-        authorization = request.META.get("HTTP_AUTHORIZATION")
+        authorization = request.META.get("HTTP_AUTHORIZATION", "").split(
```

Comment:
I think, as a more robust approach, we can keep the general check on « if authorization is empty, return None », and then partition the authorization to retrieve the keyword, and check for that separately?

```python
keyword, has_keyword, _ = authorization.partition(" ")

if not has_keyword or keyword.lower() != self.keyword.lower():
    return None
```